scipy/spatial/transform/rotation.pyx

```diff
@@ -1016,6 +1019,105 @@ cdef class Rotation(object):
         else:
             return cls(quat, normalize=False, copy=False)
 
+    @classmethod
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
+    def from_mrp(cls, mrp):
+        """Initialize from Modified Rodrigues Parameters (MRPs).
+
+        MRPs is a 3 dimensional vector co-directional to the axis of rotation and whose
+        magnitude is equal to tan(theta / 4), where theta is the angle of rotation (in radians) [1].
```

Comment:
I suggest adding double ticks (``) around "tan(theta / 4)" and "theta" to render it as code/expression.

The reference should be done as [1]_.

Reply:
Done